recipes/sigmut/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sigmut" %}
+{% set version = "1.0" %}
+{% set sha256 = "414817a3b5c7513d72226060324f093a2bcfcf96e3559a6252a053287028307c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lgpdv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - python >=3.7
+    - matplotlib-base
+    - zlib
+    - pandas
+    - numpy >=1.17.5
+    - scipy
+    - statsmodels
```

Comment:
Please delete the `host:` section and its contents.

Reply:
After deleting this section several errors have arisen. I've left the line specifying the python version and it works. However, the "test-macos" is blocked...

Reply:
You can just add `mkdir -p $PREFIX/bin` at the beginning of `build.sh` and everything will work without the `host:` section.